src/puppetlabs/trapperkeeper/app.clj

```diff
@@ -12,4 +12,5 @@
                                  "occurred, return the input parameter."))
   (init [this] "Initialize the services")
   (start [this] "Start the services")
-  (stop [this] "Stop the services"))
\ No newline at end of file
+  (stop [this] "Stop the services")
+  (restart [this] "Stop and restart the services"))
```

Comment:
This should probably return the tk to for consistency with start, stop and init.